src/AutoRest.CSharp/LowLevel/AutoRest/DpgOutputLibrary.cs

```diff
@@ -16,7 +16,7 @@ internal class DpgOutputLibrary : OutputLibrary
     {
         private readonly IReadOnlyDictionary<InputEnumType, EnumType> _enums;
         private readonly IReadOnlyDictionary<InputModelType, ModelTypeProvider> _models;
-        private readonly bool _isCadlInput;
+        private readonly bool _isTypeSpecInput;
```

Comment:
I found in some place,  tsp is used in variable names (tspOutputFileName) and in other places, typespec are used like _isTypeSpecInput.
Can we make it consistent in .ts code?